### ISSUE TITLE
Several routing prefix edits fail in a short period of time

### ISSUE THREAD

**Issue by pdine** [jeokrohn/wxc_sdk#8]
Hello,

It is not possible to define routing prefixes for several locations in a short period of time.
The aim is to be able to create and configure multiple locations at the same time.

Following function generates an exception: 
`api.telephony.location.update(location_id=location_id, settings=TelephonyLocation(routing_prefix=routing_prefix))`

This is due to a blocking API function on the Webex side that launches a Update Routing Prefix jobs: https://developer.webex.com/docs/api/v1/location-call-settings/update-location-webex-calling-details, https://developer.webex.com/docs/api/v1/location-call-settings/get-a-list-of-update-routing-prefix-jobs
It is necessary to wait until last job is finished before starting a routing prefix edit on another site.

Do you think in SDK `api.telephony.location.update` function could handle the prefix change asynchronously? This would avoid having to handle exceptions or sleep period on the script side.

**Comment by jeokrohn** [comment]
No, this is not something that can (and IMO should) be handled by the SDK. The goal of the SDK is to provide a pythonic wrapper for the existing endpoints and not to add higher level logic.

**Comment by jeokrohn** [comment]
Sorry for the delay. Looking at this issue I realized that I missed to add support for update routing prefix jobs. I added that now and the telephony location update() now returns the job id.

**Comment by jeokrohn** [comment]
There are some examples how to use the job id in the [unit tests](https://github.com/jeokrohn/wxc_sdk/blob/master/tests/test_telephony_update_routing_prefix_jobs.py).
